engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
+title: "Logentries logging driver"
+description: "Describes how to use the logentries logging driver."
+keywords: ["logentries, docker, logging, driver"]
+---
+
+# Logentries logging driver
+
+The `logentries` logging driver sends container logs to the
+[Logentries](https://logentries.com/) server.
+
+## Usage
+
+Some options are supported by specifying `--log-opt` as many times as needed:
+
+ - `logentries-token`: specify the logentries log set token
+
+Configure the default logging driver by passing the
+`--log-driver` option to the Docker daemon:
+
+```console
```

Comment:
afaik `console` is a valid highlight too, e.g.:

```console
$ ls -lha
$ sudo su
$ ps aux
$ grep -i "blah"
```

vs

```bash
$ ls -lha
$ sudo su
$ ps aux
$ grep -i "blah"
```

looks virtually the same, but it's more semantic I guess..

Anyways, I'll change them to bash... just taught there was a special reason.

Thanks =)